.github/workflows/hrm-ecs-reload-hrm.yml

```diff
@@ -0,0 +1,104 @@
+name: '[Multi Region] Reload HRM configuration via forced update'
```

Comment:
Could you remove the `[Multi Region]` prefix from this and the HRM deploy workflow names? 

It was intended to be temporary to distinguish it from a legacy workflow that was hardcoded to one region, but now that legacy workflow is gone but I keep forgetting to update the name

Reply:
Done!